src/theme/shared/allBreakpoints.js

```diff
@@ -0,0 +1,14 @@
+export const sizes = {
+  mobile: 0,
```

Comment:
Probably should have thought of this earlier, but I wonder if now would be a really good time to align this naming with what Curtis is using, in _either_ direction. I'll have a word with him and get back to you; it should be easy enough to just find-replace these if so

Reply:
Okay, to properly align with the DESIGN SYSTEM, I'm gonna suggest we go down this route:

```
export const sizes = {
  S: 0,
  M: 740,
  L: 1024,
  Nav: 1150,
  XL: 1440
};
```

Technically, Curtis uses 'SW' (SmallWeb), MW etc, but both `Nav` and the current `Large` are purely devvy stuff; the latter being used for fullwidth components.

As such, there's no `NavW` or `XLW` (both of which make me feel ill), so let's keep it cleaner as above.

Reply:
(Sorry bout the back-and-forthing :/ )